test/boilerplate.js

```diff
@@ -13,7 +13,21 @@ describe('node:boilerplate', function () {
   it('creates boilerplate files', function () {
     assert.file('lib/index.js');
     assert.file('test/index.js');
-    assert.fileContent('test/index.js', 'var myModule');
+    assert.fileContent('lib/index.js', 'export default {};');
+    assert.fileContent('test/index.js', 'import myModule');
     assert.fileContent('test/index.js', 'describe(\'my-module\'');
   });
 });
+
+describe('node:boilerplate --no-babel', function() {
```

Comment:
Spacing seems to be very [important](https://github.com/yeoman/generator-webapp/pull/543#discussion-diff-30472432) to [you](https://github.com/yeoman/generator-webapp/pull/543#discussion-diff-30472442) :stuck_out_tongue: